src/screens/LoginPage/LoginPage.tsx

```diff
@@ -133,18 +129,18 @@ function loginPage(): JSX.Element {
   const [recaptcha, { loading: recaptchaLoading }] =
     useMutation(RECAPTCHA_MUTATION);
 
-  useEffect(() => {
-    async function loadResource(): Promise<void> {
-      try {
-        await fetch(BACKEND_URL as string);
-      } catch (error: any) {
-        /* istanbul ignore next */
-        errorHandler(t, error);
-      }
-    }
-
-    loadResource();
-  }, []);
+  // useEffect(() => {
+  //   async function loadResource(): Promise<void> {
+  //     try {
+  //       await fetch(BACKEND_URL as string);
+  //     } catch (error: any) {
+  //       /* istanbul ignore next */
+  //       errorHandler(t, error);
+  //     }
+  //   }
+
+  //   loadResource();
+  // }, []);
```

Comment:
Can you please explain how can commenting this will fix the issue ?

Reply:
- Currently, that fetch request is simple request(not POST method). Any simple request to the graphql server prevented as CSRF which is why returning 400 badRequest. I tried by requesting with appropriate headers but still doesn't work.
- One other way to avoid that is using useQuery hook which is to check sever health. But that will need implementation on API side as well like `checkServerHealth` query implementation for example. 
- My solution is as there is already another query `checkAuth` which also returns same alert if negative and it is also duplicating the alert on the page, I commented this fetch request. 

Please let me know if that is not right approach @noman2002